plugins/lsp/default.nix

```diff
@@ -192,12 +192,13 @@ in
       keymapsOnEvents.LspAttach =
         let
           mkMaps =
-            prefix:
+            prefix: descPrefix:
             mapAttrsToList (
               key: action:
               let
                 actionStr = if isString action then action else action.action;
-                actionProps = if isString action then { } else filterAttrs (n: v: n != "action") action;
+                actionProps = optionalAttrs (!(isString action)) (filterAttrs (n: v: n != "action") action);
+                desc = "${descPrefix} ${actionStr}";
```

Comment:
Just to avoid confusion, perhaps name this something like `autoDesc` or `generatedDesc`?

Reply:
Or since it is only used on line 210, maybe just inline it there instead of defining it in a let-in block